Shared/Realm.Shared/Extensions/CollectionNotificationsExtensions.cs

```diff
@@ -18,20 +18,22 @@
 
 using System;
 using System.Collections.Generic;
-using System.Collections.ObjectModel;
 using System.Collections.Specialized;
 using System.Linq;
 
 namespace Realms
 {
+    /// <summary>
+    /// Realm results collection changed.
```

Comment:
We don't need this, right? Nobody will access it and this description is less than informative :-)